openwisp-config/files/openwisp.agent

```diff
@@ -575,7 +578,7 @@ restore_backup() {
 	sysupgrade -r $CONFIGURATION_BACKUP
 	pre_reload_hook
 	/usr/sbin/openwisp-reload-config
-	sleep 2
```

Comment:
not sure why I put 5 here but as a plus of this change we'll make this consistent :+1: